src/components/Modals/ExploreModals/FilterModal/FilterModal.styles.ts

```diff
@@ -1,60 +1,45 @@
 import { StyleSheet } from 'react-native';
-import { GREY_500, WHITE } from 'src/styles';
+import { title1, title2, LEVEL_1 } from 'src/styles';
+import { HORIZONTAL_12_PERCENT, HORIZONTAL_16_PERCENT } from 'src/utils';
```

Comment:
better not to use dimensions in variable names. change to PADDING_HORIZONTAL_MAIN ets.. or any suitable name